apisix/plugins/zipkin.lua

```diff
@@ -51,7 +55,12 @@ local schema = {
         span_version = {
             enum = {ZIPKIN_SPAN_VER_1, ZIPKIN_SPAN_VER_2},
             default = ZIPKIN_SPAN_VER_2,
-        },
+        },       
+        set_ngx_var = {
+            type = "boolean",
+            description = "set nginx variables",
+            default = false,
+          },
```

Comment:
Why do you add the check here? Are you going to pass this configuration in the plugin parameters?？It is unnecessary if you just want to define the configuration in `config.yaml`